plugins/changed_managed_files/changed_managed_files_inspector.rb

```diff
@@ -83,59 +82,12 @@ def amend_file_attributes(changed_files)
 
   def changed_files
     count = 0
-    list = run_script_with_progress("changed_files.sh", "--", "changed-managed-files") do |chunk|
+    file_list = @system.rpm_database.changed_files do |chunk|
       count += chunk.lines.reject { |l| l.chomp.end_with?(":") || l.split(" ")[1] == "c" }.count
       Machinery::Ui.progress(" -> Found #{count} changed #{Machinery::pluralize(count, "file")}...")
+    end.reject { |f| f.config_file? }.map do |file|
```

Comment:
Avoid multi-line chains of blocks.<br>Pass `&:config_file?` as an argument to `reject` instead of a block.